### DISCUSSION TITLE
Use import file extensions

### DISCUSSION THREAD

**Discussion by sachaw** [chakra-ui/ark#2294] [Ideas]
### Description

Hi, is there any opposition to using file extensions for imports & exports:
https://www.totaltypescript.com/relative-import-paths-need-explicit-file-extensions-in-ecmascript-imports
This is required when using the modern node module resolution strategy.

### Link to Reproduction (or Detailed Explanation)

NIL

### Steps to Reproduce

NIL

### Ark UI Version

NIL

### Framework

- [ ] React
- [ ] Solid
- [ ] Vue

**Comment by cschroeter** [comment]
@sachaw

Thank you for raising this issue. However, I'm not sure why this change is necessary. As far as I know, the way a library handles its imports and exports should not impact the end-user's application.
